core/src/main/java/com/sequenceiq/cloudbreak/service/template/ClusterTemplateViewService.java

```diff
@@ -37,7 +37,7 @@ protected void prepareCreation(ClusterTemplateView resource) {
         return repository.findAllActive(workspaceId);
     }
 
-    public Set<ClusterTemplateView> findAllByEnvironmentCrn(String environmentCrn) {
-        return repository.getAllByEnvironmentCrn(environmentCrn);
+    public Set<ClusterTemplateView> findAllByEnvironmentCrn(String environmentCrn, String cloudPlatform, String runtime) {
```

Comment:
The name should describe that this will return only with non deleted defaults by optional cloudplatform and runtime as the repository call too.